File: src/CustomerBuilderInterface.java
public interface CustomerBuilderInterface {
    CustomerBuilderInterface firstName(String firstname);
    CustomerBuilderInterface middleName(String middleName);
    CustomerBuilderInterface lastName(String lastName);
    CustomerBuilderInterface primaryEmail(String primaryEmail);
    CustomerBuilderInterface secondaryEmail(String secondaryEmail);
    CustomerBuilderInterface mobile(String mobile);
    CustomerBuilderInterface address(String address);
}

File: src/CustomerBuilderInterface.kt
interface CustomerBuilderInterface {
    fun firstName(firstname : String): CustomerBuilder
    fun middleName(middleName : String): CustomerBuilder
    fun lastName(lastName : String): CustomerBuilder
    fun primaryEmail(primaryEmail : String): CustomerBuilder
    fun secondaryEmail(secondaryEmail : String): CustomerBuilder
    fun mobile(mobile : String): CustomerBuilder
    fun address(address : String): CustomerBuilder
}
